### PR TITLE
docs(react-native): UI component - CallContentView

### PR DESCRIPTION
**Pull request by khushal87**

### CODE REVIEW

**Comment by santhoshvai** [comment]
The screenshots attached are not how it looks currently. I kindly suggest we both go on a call and put a nice screenshot here.

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx [hunk 1]
```diff
@@ -3,7 +3,47 @@ id: call-content-view
 title: CallContentView
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
+CallContentView is one of the main components for a Call. This is responsible for listing all the participant's video stream in a vertical/horizontal list and play the audio streams for those participants.
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx:6]:
```suggestion
CallContentView is one of the main components for a Call. This is responsible for listing all the participant's video streams in a vertical/horizontal list and playing the audio streams for those participants.
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx [hunk 2]
```diff
@@ -3,7 +3,47 @@ id: call-content-view
...
+
+![Preview of the CallContentView component.](../../assets/04-ui-components/call-content-view-grid.png)
+
+The CallContentView has two modes(`grid` or `spotlight`). The grid mode simply lists the participants vertically one after the other, while the spotlight mode is responsible to show the screen sharing view, a single dominant speaker, or a pinned participant in an elongated view, while other participants being in an horizontal view.
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx:10]:
```suggestion
The CallContentView has two modes(`grid` or `spotlight`). The grid mode lists the participants vertically, one after the other. In contrast, the spotlight mode shows the screen sharing view, or a single dominant speaker, or a pinned participant in an elongated view, while other participants are in a horizontal view.
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx [hunk 3]
```diff
@@ -3,7 +3,47 @@ id: call-content-view
...
+
+```tsx
+import {
+  Call,
+  CallContentView,
+  StreamCall,
+} from '@stream-io/video-react-native-sdk';
+
+const VideoCallUI = () => {
+  let call: Call;
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx:26]:
```suggestion
  let call: Call;
  // your logic to create or getting an existing call 
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx [hunk 4]
```diff
@@ -3,7 +3,47 @@ id: call-content-view
...
+
+## Props
+
+### `mode`
+
+| Type                  | Default Value |
+| --------------------- | ------------- |
+| `grid` \| `spotlight` | `grid`        |
+
+This is used to switch the list between the grid and the spotlight mode.
```
Comment (santhoshvai) [inline packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/call/call-content-view.mdx:45]:
```suggestion
This switches the list between the grid and the spotlight mode.
```

**Comment by khushal87** [comment]
@santhoshvai I have added some new screenshots

**Pull request merged**